### PR TITLE
Update teams-skype-interop.md

### PR DESCRIPTION
**Pull request by lindspea**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/5102
Removed information

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/teams-skype-interop.md [hunk 1]
```diff
@@ -56,7 +56,6 @@ Blocked Skype users, along with other people and public switched telephone netwo
 
 - Conversations are text-only. This means that there's no rich formatting, @mentions, emojis, or other any of the other chat features that are available in a [native Teams chat experience](native-chat-for-external-users.md).
```
Comment (JohanFreelancer9) [inline Teams/teams-skype-interop.md:57]:
```suggestion
- Conversations are text-only. This means that there's no rich formatting, @mentions, emojis, or any of the other chat features that are available in a [native Teams chat experience](native-chat-for-external-users.md).
```

**Comment by scanum** [comment]
@LanaChin PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by LanaChin** [comment]
Confirmed with PM. Statement is correct. Team users and Skype users can't see each other's presence. This is not supported. Closing this pull request without merging.

**Pull request closed without merging**